docs/content/preview/architecture/docdb-sharding/tablet-splitting.md

```diff
@@ -167,79 +165,74 @@ Imagine there is a table with pre-existing data spread across a certain number o
+
 ### Manually split the tablet
 
-The tablet of this table can be manually split into two tablets by running the following [`yb-admin split_tablet`](../../../admin/yb-admin/#split-tablet) command.
+The tablet of this table can be manually split into two tablets by running the following [`yb-admin split_tablet`](../../../admin/yb-admin/#split-tablet) command:
 
 ```sh
-bin/yb-admin \
+.bin/yb-admin \
     --master_addresses 127.0.0.1:7100,127.0.0.2:7100,127.0.0.3:7100 \
     split_tablet 9991368c4b85456988303cd65a3c6503
 ```
 
-After the split, you see two tablets for the table `t`.
-
-```sh
-bin/yb-admin --master_addresses 127.0.0.1:7100 list_tablets ysql.yugabyte t
-```
+After the split, you expect to see two tablets for the table `t`, as per the following output:
 
 ```output
 Tablet UUID                       Range                                 Leader
 20998f68c3fa4d299e8af7c04410e230  key_start: "" key_end: "\177\377"     127.0.0.1:9100
 a89ecb84ad1b488b893b6e7762a6ca2a  key_start: "\177\377" key_end: ""     127.0.0.3:9100
 ```
 
-{{< note title="Important" >}}
-
-* The original tablet `9991368c4b85456988303cd65a3c6503` no longer exists and has been replaced with two new tablets.
-* The tablet leaders are now spread across two nodes in order to evenly balance the tablets for the table across the nodes of the cluster.
+The original tablet `9991368c4b85456988303cd65a3c6503` no longer exists; it has been replaced with two new tablets.
 
-{{< /note >}}
+The tablet leaders are now spread across two nodes in order to evenly balance the tablets for the table across the nodes of the cluster.
 
 ## Automatic tablet splitting
 
-Automatic tablet splitting enables resharding of data in a cluster automatically while online, and transparently to users, when a specified size threshold has been reached.
+Automatic tablet splitting enables resharding of data in a cluster automatically while online, as well as transparently, when a specified size threshold has been reached.
 
-For details on the architecture design, see [Automatic Re-sharding of Data with Tablet Splitting](https://github.com/yugabyte/yugabyte-db/blob/master/architecture/design/docdb-automatic-tablet-splitting.md).
+For details on the architecture design, see [Automatic re-sharding of data with tablet splitting](https://github.com/yugabyte/yugabyte-db/blob/master/architecture/design/docdb-automatic-tablet-splitting.md).
 
 ### Enable automatic tablet splitting
 
-To enable automatic tablet splitting, use the `yb-master` [`--enable_automatic_tablet_splitting`](../../../reference/configuration/yb-master/#enable_automatic_tablet_splitting) command, and specify the associated flags to configure when tablets should split.
+To enable automatic tablet splitting, use the `yb-master` [`--enable_automatic_tablet_splitting`](../../../reference/configuration/yb-master/#enable_automatic_tablet_splitting) command and specify the associated flags to configure when tablets should split.
```

Comment:
yb-master/#enable-automatic-tablet-splitting
also, on the yb-master page, these are referred to as flags

Reply:
Fixed